### PR TITLE
Move professional learning marketing pages into /professional-learning directory

### PR DESCRIPTION
**Pull request by kelbyhawn**
Moves and redirects pages in the `/educate/professional-learning` directory into the new `/professional-learning` directory. The new page paths are:
- [code.org/educate/professional-learning](https://code.org/educate/professional-learning) → [code.org/professional-learning](https://code.org/professional-learning)
- [code.org/educate/professional-development-online](https://code.org/educate/professional-development-online) → [code.org/professional-learning/self-paced](https://code.org/professional-learning/self-paced)
- [code.org/educate/professional-learning/contact-regional-partner](https://code.org/educate/professional-learning/contact-regional-partner) → [code.org/professional-learning/contact-regional-partner](https://code.org/professional-learning/contact-regional-partner)
- [code.org/educate/professional-learning/facilitator](https://code.org/educate/professional-learning/facilitator) → [code.org/professional-learning/facilitator](https://code.org/professional-learning/facilitator)
- [code.org/educate/professional-learning/middle-high](https://code.org/educate/professional-learning/middle-high) → [code.org/professional-learning/middle-high](https://code.org/professional-learning/middle-high)
- [code.org/educate/professional-learning/values](https://code.org/educate/professional-learning/values) → [code.org/professional-learning/values](https://code.org/professional-learning/values)

**Jira ticket:** [ACQ-1541](https://codedotorg.atlassian.net/browse/ACQ-1541)

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/professional-learning/values.md.erb
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/professional-learning/values.md.erb:1]: Only whitepsace changes here.

File: pegasus/sites.v3/code.org/public/professional-learning.moved
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/professional-learning.moved:1]: Deleted this since we now have https://code.org/professional-learning/index

**Review by bethanyaconnor — APPROVED** [review]

**Review by TurnerRiley — COMMENTED** [review]
Does [this UI test file](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/features/acquisition_products/pd/regional_partner_mini_contact.feature) need to be updated since it references `/educate/professional-learning/contact-regional-partner`?

**Comment by kelbyhawn** [comment]
> Does [this UI test file](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/features/acquisition_products/pd/regional_partner_mini_contact.feature) need to be updated since it references `/educate/professional-learning/contact-regional-partner`?

@TurnerRiley ooo good thinking! Just updated 👍

**Pull request merged**